orangewidget/gui.py

```diff
@@ -2077,6 +2076,8 @@ def do_commit():
         setattr(master, 'unconditional_' + commit_name, commit)
         setattr(master, commit_name, conditional_commit)
 
+    checkbox_toggled()
```

Comment:
This must be moved here, otherwise a widget that inherits an autocommit and does not redecorate the commit function will fall with uninformative `AttributeError` in `checkbox_toggled` before failing with `RuntimeError` that is triggered above.